mustard_ui.py

```diff
@@ -1457,10 +1464,12 @@ def execute(self, context):
             for j in range(len(mat.node_tree.nodes)):
                 if "MustardUI Float" in mat.node_tree.nodes[j].name and mat.node_tree.nodes[j].type=="VALUE":
                     mustardui_add_option_item_body(rig_settings.body_additional_properties, [mat.node_tree.nodes[j].name[len("MustardUI Float - "):], mat.node_tree.nodes[j].outputs[0].default_value, 3])
+                elif "MustardUI BigFloat" in mat.node_tree.nodes[j].name and mat.node_tree.nodes[j].type=="VALUE":
+                    mustardui_add_option_item_body(rig_settings.body_additional_properties, [mat.node_tree.nodes[j].name[len("MustardUI BigFloat - "):], mat.node_tree.nodes[j].outputs[0].default_value, 4])

```

Comment:
Also need to change the mustardui_add_option_item_body in line 1417.

Reply:
Good catch, update made 👍